server/webapp/WEB-INF/rails.new/app/views/layouts/pipeline_configs.html.erb

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html class="no-js">
```

Comment:
Is this supposed to be added here?

This is not valid HTML according to 
https://www.w3.org/TR/html4/struct/global.html#h-7.3